### PR TITLE
NEWS: more reformulations and fixlets

### PR DESCRIPTION
**Pull request by behrmann**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -66,17 +66,18 @@ CHANGES WITH 256 in spe:
 
         * A new 'systemd-vpick' binary has been added. It implements the new
           vpick protocol, where a .v directory may contain multiple files with
-          a version embedded in the name, and they may be ordered by version
-          and the newest one can be reliably selected.
+          a version following the UAPI version format specification the
+          embedded in the name. The files are ordered by version and the newest
```
Comment (YHNdnzj) [inline NEWS]: Can't interpret this?

Comment (behrmann) [reply NEWS]: Removed the superfluous "the" before embedded and added commas for added clarity.

**Review by YHNdnzj — APPROVED** [review]

File: NEWS [hunk 2]
```diff
@@ -295,8 +297,8 @@ CHANGES WITH 256 in spe:
         Systemd-boot and systemd-stub and Related Tools:
 
         * TPM 1.2 PCR measurement support has been removed from systemd-stub.
-          TPM 1.2 is obsolete and — due to the (by today's standards) weak
-          cryptographic algorithms it only supports — does not actually provide
+          TPM 1.2 is obsolete and—due to the (by today's standards) weak
+          cryptographic algorithms it only supports—does not actually provide
```
Comment (keszybz) [inline NEWS]: Please don't do this. I know it's the usual English style, but it makes no sense. It's pretty bad with a real font, where the long dash can be made wide, but it's even worse with a monospace font where the long dash is displayed as a normal dash. NEWS is supposed to be readable with a monospace font.

Comment (behrmann) [reply NEWS]: Pushed a version using an en dash instead of an em dash, which can be used in the same way and is typeset with spaces around.

**Pull request merged**